src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -43,15 +43,50 @@ public void testVCFModeIsConsistentWithPastResults() throws Exception {
         IntegrationTestSpec.assertEqualTextFiles(output, expected);
     }
 
+
     /*
-     * Test that in VCF mode we're >= 99% concordant with GATK3.4 results
+     * Test that in VCF mode we're consistent with past GATK4 results
      */
     @Test
-    public void testVCFModeIsConcordantWithGATK3Results() throws Exception {
+    public void testVCFModeIsConsistentWithPastResults_AlleleSpecificAnnotations() throws Exception {
+        Utils.resetRandomGenerator();
+
+        //NOTE: AlleleSpecific support in the VCF mode is implemented but bous for now.
+        // This test should not be treated as a strict check of correctness.
+        final File output = createTempFile("testVCFModeIsConsistentWithPastResults", ".vcf");
+        final File expected = new File(TEST_FILES_DIR + "expected.testVCFMode.gatk4.alleleSpecific.vcf");
+
+        final String[] args = new String[] {
+                "-I", NA12878_20_21_WGS_bam,
+                "-R", b37_reference_20_21,
+                "-L", "20:10000000-10100000",
+                "-O", output.getAbsolutePath(),
+                "-G", "StandardAnnotation",
+                "-G", "AS_StandardAnnotation",
+                "-pairHMM", "AVX_LOGLESS_CACHING"
+        };
+
+        runCommandLine(args);
+
+        // Test for an exact match against past results
+        IntegrationTestSpec.assertEqualTextFiles(output, expected);
+    }
+
+    /*
+     * Test that in VCF mode we're >= 99% concordant with GATK3.5 results
+     */
+    @Test
+    public void testVCFModeIsConcordantWithGATK3_5Results() throws Exception {
         Utils.resetRandomGenerator();
 
         final File output = createTempFile("testVCFModeIsConcordantWithGATK3Results", ".vcf");
-        final File gatk3Output = new File(TEST_FILES_DIR, "expected.testVCFMode.gatk3.4.vcf");
```

Comment:
You left the previous expected output `expected.testVCFMode.gatk3.4.vcf` checked into the test directory -- should we delete it, or is it worth keeping around for a while for troubleshooting purposes?